src/recap.js

```diff
@@ -169,18 +170,54 @@ function Recap() {
           processData: false,
           contentType: false,
           data: formData,
-          success: function(data, textStatus, xhr){
+          success: function (data, textStatus, xhr) {
             console.info(`RECAP: Successfully uploaded PDF: '${textStatus}' ` +
-                        `with processing queue id of ${data['id']}`);
+              `with processing queue id of ${data['id']}`);
             cb(data || null);
           },
-          error: function(xhr, textStatus, errorThrown){
+          error: function (xhr, textStatus, errorThrown) {
             console.error(`RECAP: Ajax error uploading PDF. Status: ${textStatus}.` +
-                          `Error: ${errorThrown}`);
+              `Error: ${errorThrown}`);
           }
         });
 
       });
+    },
+    // Upload a zip file to the RECAP server, calling the cb with ok flag
+    uploadZipFile: async function (pacer_court, pacer_case_id, nonce, cb) {
+      console.info(`RECAP: Attempting PDF upload to RECAP Archive with details: ` +
+        `pacer_court: ${pacer_court}, pacer_case_id: ${pacer_case_id}`)
+
+      // async call for the blob
+      const bytes = await getItemsFromStorage([nonce, 'docId'])
```

Comment:
`bytes` seems like a funny variable name here, no?

Reply:
what would you prefer?

Reply:
Well, it's a dict with a couple entries, so something like storage_response might make sense? Or is there a clever way to do something like this in JS:

```
const bytes, docId = await getItemsFromStorage([nonce, 'docId'])
```

But the larger point was that the response from `getItemsFromStorage` isn't just a big blob of data, it's a dictionary with a few things in it. I was surprised to see that bytes had keys a few lines later.

Reply:
I see you've tweaked this. Seems better, thanks.